src/runtime_src/core/edge/user/device_linux.cpp

```diff
@@ -1130,6 +1130,13 @@ set_cu_read_range(cuidx_type cuidx, uint32_t start, uint32_t size)
     throw xrt_core::error(ret, "failed to set cu read range");
 }
 
+xclGraphHandle
+device_linux::
+open_graph(const xrt::hw_context& hwctx, const char* name, xrt::graph::access_mode am)
+{
+   return xclGraphOpen(get_device_handle(), hwctx.get_xclbin_uuid().get(), name, am);
+}
+
```

Comment:
Remove, not needed

```suggestion
```

Reply:
Same here. Just to make the functionality correct for single hardware context flows, This API is calling old API with uuid. But this implementation would be changed to handle/make use of hardware context.